cmd/experimental/kueue-viz/frontend/.env

```diff
@@ -1 +1,2 @@
+VITE_WEBSOCKET_URL=ws://localhost:8080
 REACT_APP_WEBSOCKET_URL=ws://localhost:8080
```

Comment:
Can we use only one?

Reply:
theorically, we can. But, I prefer to defer that one to another PR:
- as a general practice, React apps expects their env var to start with REACT_APP and same for Vite apps. But, as we are migrating, and for compatibility purposes I want to let both of them to avoid breaking users.
- maybe we should get rid off the prefix or use a tech agnostic prefix like KUEUE_VIZ_